Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Process_attempts_to_kill_a_known_security_or_AV_tool.yml

```diff
@@ -0,0 +1,992 @@
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 99b56c83-2667-4359-8d0c-72ee7119c91e
+    type: condition
+    task:
+      id: 99b56c83-2667-4359-8d0c-72ee7119c91e
+      version: -1
+      name: Found related alerts indicating malicious activity?
+      description: Checks whether found related alerts indicating malicious activity.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "49"
+      "yes":
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 761,
+          "y": 739
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: ed7584b7-7a5d-4347-8e42-66d1651af01f
+    type: regular
+    task:
+      id: ed7584b7-7a5d-4347-8e42-66d1651af01f
+      version: -1
+      name: Close Alert
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "34"
+    scriptarguments:
+      closeNotes:
+        simple: Resolved as True Positive - Handled by the playbook "Process attempts to kill a known security or AV tool"
+      closeReason:
+        simple: Resolved - True Positive
+      id:
+        complex:
+          root: alert
+          accessor: id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 420,
+          "y": 2220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: b1e77e68-7d8a-46ee-8098-656e282f6345
+    type: title
+    task:
+      id: b1e77e68-7d8a-46ee-8098-656e282f6345
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 420,
+          "y": 227
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: dbaedd9f-d40d-4ff0-bd7a-081349607692
+    type: condition
+    task:
+      id: dbaedd9f-d40d-4ff0-bd7a-081349607692
+      version: -1
+      name: Is the endpoint risk score high?
```

Comment:
please validate that the key exist as the first condition
`Core.RiskyHost.risk_level` 
(as the get host risk score task is set to contuie on error)

Reply:
Done